tests/foreman/cli/test_import.py

```diff
@@ -1678,6 +1681,18 @@ def test_negative_reimport_config_files(self):
                     Repository.info({'id': rec['sat6']})
                     for rec in imp_configs
                 ]
+                # sequential import
+                import_cf = Import.config_file_with_tr_data({
+                    'csv-file': files['config-files-latest'],
+                    'verbose': True,
+                })
+                configs = Import.csv_to_dataset([files['config-files-latest']])
+                imp_configs = get_sat6_id(
+                    configs,
+                    import_cf[1][1],
+                    'channel_id',
+                    'channel_id'
+                )
```

Comment:
I know, he knows,
(he told me then that he was forced by others to name it so...)

it tests that nothing has been imported by a sequential run (idempotency test)